classes/migration/install/CommonMigration.php

```diff
@@ -96,6 +96,7 @@ public function up(): void
 
         switch (DB::getDriverName()) {
             case 'mysql':
+            case 'mariadb':
```

Comment:
In situations where there's a `case` statement running through all the possible drivers, please make sure there is also a:
```php
   default: throw new \Exception('Unexpected database driver!');
```
Also elsewhere.